src/app/lamps/thing_broadcast.py

```diff
@@ -0,0 +1,44 @@
+from machine import Pin
+from ..lamp_core.base_lamp import BaseLamp
+from ..network.coding import Codes
+from ..colors import *
+
+class ThingBroadcast(BaseLamp):
```

Comment:
What makes this Thing specific (vs just "Broadcast" ?

Reply:
This is for the `ESP32Thing`. It has a button on it which is non-standard. I mostly just use this to do broadcast tests.

Reply:
aha